data_models/asana_data_model.py

```diff
@@ -0,0 +1,16 @@
+import panther_event_type_helpers as event_type
+
+
+def get_event_type(event):
+    audit_log_type_map = {
+        "user_login_succeeded": event_type.SUCCESSFUL_LOGIN,
+        "user_login_failed": event_type.FAILED_LOGIN,
+        "user_invited": event_type.USER_ACCOUNT_CREATED,
+        "user_reprovisioned": event_type.USER_ACCOUNT_CREATED,
+        "user_deprovisioned": event_type.USER_ACCOUNT_DELETED,
+        "user_workspace_admin_role_changed": event_type.ADMIN_ROLE_ASSIGNED,
+    }
+    logged_event_type = event.get("event_type")
```

Comment:
Should this return an empty dict by default?

```suggestion
    logged_event_type = event.get("event_type", {})
```

Reply:
👍🏼  Good call, updated